libraries/freertos_plus/standard/freertos_plus_tcp/include/FreeRTOSIPConfigDefaults.h

```diff
@@ -314,7 +314,11 @@ from the FreeRTOSIPConfig.h configuration header file. */
 #endif
 
 #ifndef ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND
-	#define ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND 1
+	#define ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND	1
```

Comment:
Is this documented anywhere?  Why would anyone want to set `ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND==0`?

Reply:
This macro already existed in the UDP-only library, I only changed white-space.

If the macro is not defined, calling `sendto()` with an unbound socket will fail.

>  Why would anyone want to set `ipconfigALLOW_SOCKET_SEND_WITHOUT_BIND==0`?

Good question. I don't see an advantage of forcing programmers to bind a socket explicitly.